graphics/h3dapi/Portfile

```diff
@@ -27,23 +28,27 @@ subport             h3dviewer {
 }
 
 homepage            http://www.h3dapi.org
-master_sites        ftp://www.h3dapi.org/pub/releases/source
+master_sites        ftp://www.h3dapi.org/pub/releases/source/
 
-checksums           rmd160  ca60af0fcd89c309cd030ab509a640b28fd6b470 \
-                    sha256  231104455feea6217c103ab0cbb2187e448000e6f02f9a1f2abb0161a13bbbd1
+checksums           rmd160  4bfbba220a9985f87c258d9ae191bc54e55bbd30 \
+                    sha256  60e82718be851b63b7cbd96d57ff6d0917aed39be98bb287bb42871aa5cc6f10
 
 cmake.out_of_source yes
 
-patchfiles          patch-CMakeLists.txt.diff \
-                    patch-include-H3D-X3DParticleEmitterNode.h.diff \
-                    patch-CMakeListsPython.diff \
-                    patch-glutwindow.diff \
-                    patch-h3dapicmake.diff \
-                    patch-pythonmethods.diff \
-                    patch-pythonscript.diff \
-                    patch-pythontypes.diff \
-                    patch-pytypewrapper.diff \
-                    patch-pythontypesheader.diff
+depends_lib         port:hapi \
+                    port:glew \
+                    port:xercesc \
+                    port:ftgl \
+                    port:libvorbis \
+                    port:audiofile \
+                    port:spidermonkey \
+                    port:fontconfig \
+                    port:freeglut \
+                    port:freetype \
+                    port:bzip2 \
+                    port:curl \
+                    port:python27 \
+                    port:openexr
```

Comment:
Please sort these alphabetically.

Reply:
Can python37 be used instead of the soon to be deprecated python27?

Reply:
H3DAPI and the other toolkits comes with a bunch of examples (well the source code does). These examples have not yet been tested and fixed for python 3 and as such we can not use python 3 by default.
The python 3 support is in general still not throughly tested and it would be up to any user to link against that.
When you say deprecated do you refer to 2020 or will you deprecate it on macports earlier?

Reply:
Besides, anyone that has used H3D before will have unworkable projects if we use python 3 as default. Since it was quite some time since H3D last was released it is better to release one more version about python 2.

Reply:
I'm talking about upstream support for python 2.7. If this project is not ready for python 3.0 yet, then definitely keep the dependency on python 2.7.